Bermuda/Bermuda/InteractableNPC.cpp

```diff
@@ -0,0 +1,78 @@
+#include "InteractableNPC.h"
+
+InteractableNPC::InteractableNPC(int id, int healthPoints, int attackPoints, int actionRange, int attackRange, Spawnpoint *spawnPoint, int interactStartX, int interactStartY, int interactWitdh, int interactHeight, long attackTime) :
+	NPC( id, actionRange, attackRange, spawnPoint ),
+	Entity( id, spawnPoint->getX(), spawnPoint->getY() ),
+	InteractableEntity(id, spawnPoint->getX(), spawnPoint->getY(), interactStartX, interactStartY, interactWitdh, interactHeight ),
+	AttackingNPC(attackTime)
```

Comment:
Persoonlijk zou ik het toch gescheiden houden denk ik. Juist omdat NPC's niet altijd attacking NPC's hoeven te zijn. In het geval van de konijnen in ieder geval, maar ook als je rekening houd dat altijd NPC's toegevoegd kunnen worden die niet aanvallen. (rekening houden met de toekomst)